xclim/indicators/atmos/_temperature.py

```diff
@@ -307,7 +310,17 @@ def cfcheck(tasmin, tasmax):
     long_name="Mean Diurnal Temperature Range",
     description="{freq} mean diurnal temperature range.",
     cell_methods="time range within days time: mean over days",
-    compute=indices.daily_temperature_range,
+    compute=wrapped_partial(indices.daily_temperature_range, op="mean"),
+)
+
+max_daily_temperature_range = TasminTasmax(
```

Comment:
I think we should explicitly declare the "title" field of both dtr indicators. Otherwise, the generic "statistics of daily ..." will appear (twice).

Reply:
noted - good catch.. will add that to the branch

Reply:
Done